ng/js/directives/header.js

```diff
@@ -0,0 +1,36 @@
+require('angular').module('ays')
+
+  /* Author: Tyler Russell
+   * Directive to be placed in various HTML headers for user management,
+   * login/logout navigation, etc.
+   */
+  .directive('sessionheader', function (AuthService, UserService) {
+    return {
+      restrict: 'E',
+      controller: function ($scope, $sails) {
```

Comment:
Annotate in angular refers to explicitly defining the providers being injected instead of relying on Angular inferring from variable name: ['$scope', function($scope){...}]. Makes it safe to minify the code